app/main.py

```diff
@@ -1,3 +1,57 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int or float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km!r})"
+
+    def __add__(self, other: int or "Distance") -> "Distance":
```

Comment:
Indicate that other could be `float` as well. Do not use `""` in the type annotation 
```suggestion
    def __add__(self, other: int | float | Distance) -> Distance:
```
In case you face errors, use `from __future__ import annotations` import